### ISSUE TITLE
bash aliases at Terminal

### ISSUE THREAD

**Issue by TMTYD** [webmin/webmin#717]
Hi

When I open the terminal (command line) in the Webmin interface (ALT + K), is it possible to use its own bash aliases?

**Comment by jcameron** [comment]
No, aliases aren't supported as it isn't a real bash shell at all. Instead, it just runs commands one by one on the server.

**Comment by TMTYD** [comment]
ok, actually, so you could use his personal aliases, which you otherwise also use in the terminal when you connect via Putty
